common/djangoapps/util/milestones_helpers.py

```diff
@@ -367,6 +367,9 @@ def get_course_content_milestones(course_id, content_id, relationship, user_id=N
             user={"id": user_id}
         )
 
+    if content_id is None:
```

Comment:
Since `content_id` is now an optional parameter, can you update the method signature to make this clear?  Here are 2 options:

1. Create a new method called `get_course_milestones` that doesn't take a `content_id` and distinguishes from this method that does.  For DRY-sake, move the bulk of this code into a new internal helper that both public methods call.

2. Change the signature to `get_course_content_milestones(course_id, content_id=None, relationship='requires', user_id=None)` and update the docstring so it's clear that `content_id` is optional.

Reply:
That's cleaner, I'll incorporate that change